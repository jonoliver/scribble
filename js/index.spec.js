import React from 'react';
import { mount } from 'enzyme';
import Game from '.';
import renderer from 'react-test-renderer';

const worker = {
  onMessage: () => {},
  postMessage: () => {},
}

it('renders', () => {
  const tree = renderer
    .create(<Game {...{ worker }} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});