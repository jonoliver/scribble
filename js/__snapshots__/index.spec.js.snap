// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
Array [
  <div
    className="instructions"
  >
    <h1>
      Scribble
    </h1>
    <h2>
      A fun game to hone your wordmaking ability
    </h2>
    <p>
      Drag letters into the white area to create words. You have you have 
      45
       seconds to beat the robot!
    </p>
  </div>,
  <select
    onChange={[Function]}
    value={1}
  >
    <option
      value="2"
    >
      Easy
    </option>
    <option
      value="1"
    >
      Medium
    </option>
    <option
      value="0"
    >
      Hard
    </option>
  </select>,
  <button
    id="start"
    onClick={[Function]}
  >
    Play
  </button>,
]
`;
